codex-ui/src/vue/components/form/Fieldset.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div :class="[$style['fieldset'], $style['fieldset__title'], 'text-ui-large']">
+    {{ title }}
+  </div>
+  <slot />
```

Comment:
there is also a wrapper with gap of "30px" in figma 

<img width="1691" alt="image" src="https://github.com/codex-team/notes.web/assets/3684889/d3297423-c3ac-493e-ae75-1c9977ec652f">